.circleci/config.yml

```diff
@@ -342,9 +342,11 @@ steps-mksnapshot-build: &steps-mksnapshot-build
         command: |
           cd src
           ninja -C out/native_mksnapshot v8:mksnapshot
+          ninja -C out/native_mksnapshot electron:electron_mksnapshot_zip
+          cp out/mksnapshot/mksnapshot.zip out/native_mksnapshot/mksnapshot_native.zip
```

Comment:
How is `out/mksnapshot/mksnapshot.zip` created? You're building in `out/native_mksnapshot`.

Reply:
whoops reversed it: should be `cp out/native_mksnapshot/mksnapshot_native.zip out/mksnapshot/mksnapshot.zip`

Reply:
@codebytere no it should be ```cp out/native_mksnapshot/mksnapshot.zip out/native_mksnapshot/mksnapshot_native.zip``` because ```ninja -C out/native_mksnapshot electron:electron_mksnapshot_zip``` generates the file ```out/native_mksnapshot/mksnapshot.zip```